README.md

```diff
@@ -1,23 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Sewing Pattern Maker
 
-## Available Scripts
+Our web app allows users to generate a sewing pattern from a  photo of an article of clothing.
```

Comment:
This is silly, but I think there is an extra space in between "a" and "photo" "from a  photo"

Reply:
good catch.